test/end-to-end/utils/helper.js

```diff
@@ -8,15 +8,15 @@ module.exports = {
     if (pageConfig.root.includes('9090')) {
       nightmare
         .goto(page.url)
-        .wait(page.iframeLoadingTime)
+        .wait(page.iframeSelector)
         .enterIFrame(page.iframeSelector);
     } else {
       nightmare
         .goto(page.url);
     }
   },
   gotoError: (error, nightmare, testSpec) => {
-    console.error(`Failed on case ${testSpec.result.fullName} - ${error}`);
+    console.error(`Failed on case ${testSpec.result.fullName} - ${error.toString()} - ${JSON.stringify(error)}`);
```

Comment:
Why "${JSON.stringify(error)"?
From the result log, this did not work here.
Failed on case View Blueprint Page Single Word Blueprint Name Scenario Menu Nav Bar Check should show a blueprint name - Error: .wait() timed out after 5000msec - {}

Reply:
When testing with lorax-composer Cockpit is served via https and initially I was getting an error because of untrusted SSL certificate. Without JSON.stringify the only thing I was shown was [object Object]. Below is an example with both error logging methods enabled:

```
  console.error utils/helper.js:19
    Failed on case View Blueprint Page Single Word Blueprint Name Scenario Menu Nav Bar Check should show a blueprint name - [object Object] - {"message":"navigation error","code":-501,"details":"ERR_INSECURE_RESPONSE","url":"https://localhost:9090/welder#/blueprint/automation"}
```

In other cases, like you point out the error object is logged as string and JSON.stringify returns an empty dictionary. I think this is due to how these error objects are created but this isn't something under our control. That's why I've added both logging methods hoping to be able to get the most error information as possible.

Reply:
Make sense to me. Thanks @atodorov.